lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -3646,6 +3647,12 @@ LogicalResult FIRRTLLowering::visitExpr(PlusArgsTestIntrinsicOp op) {
   return setLoweringTo<sv::ReadInOutOp>(op, reg);
 }
 
+LogicalResult FIRRTLLowering::visitExpr(FPGAProbeIntrinsicOp op) {
+  auto operand = getLoweredValue(op.getInput());
+  operand = builder.create<hw::WireOp>(operand);
```

Comment:
There's no reason to bind the return value here since the wire remains unused. In general, avoid re-assignment.